### DISCUSSION TITLE
Github Copilot course not advancing to Step 3 after successfully completing step 2

### DISCUSSION THREAD

**Discussion by isohaildevoutlook** [skills/.github#304] [Code with Copilot]
After completing the step 2 on CoPilot course, it doesn't advance to step3. The workflow fails to complete and no information is available. Stuck, unable to proceed. Please help.

**Comment by sinsukehlab** [comment]
See the errors on your Actions tab.
The workflow "2-skills-javascript.yml" greps the "skills.js" file for "function calculateNumbers" (case-sensitive).
The workflow "3-copilot-hub.yml" greps the "member.js" file for "skillsMember" (case-sensitive).
The workflow "4-copilot-comment.yml" greps the "comments.js" file for "Create web server" (case-sensitive).
Cf. #286
